api/src/main/java/org/openmrs/module/ehraddons/reports/SetupChildrenLabResultsReport.java

```diff
@@ -38,10 +38,10 @@
 	@Override
 	protected List<Mapped<DataSetDefinition>> buildDataSets(ReportDescriptor descriptor, ReportDefinition report) {
 		SqlDataSetDefinition dsd = new SqlDataSetDefinition();
-		dsd.setName("CHILDREN");
+		dsd.setName("labchildren");
 		dsd.addParameter(new Parameter("startDate", "Start Date", Date.class));
 		dsd.addParameter(new Parameter("endDate", "End Date", Date.class));
-		dsd.setSqlQuery(LabReportQueries.getLabChildrenResultsQuery());
+		dsd.setSqlQuery("SELECT * FROM patient LIMIT 100");
```

Comment:
Why not supply the right query for this? we need when data is entered. we get results